### PR TITLE
Link occa-transpiler as a library

### PR DESCRIPTION
**Pull request by thilinarmtb**
## Description

This is required when using a packaged version of occa-transpiler (for example from `conda`).

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -76,13 +74,7 @@ When cross compiling for a different platform, the targeted hardware doesn't nee
 
 #### Building with Clang transplier option
 
-
-Hard dependency is clang-17 exactly. So far clang based transpiler does not have compatibility layer to support differences of C++ API in clang tooling in newer versions. How to install it please refer to the original 
-[clang occa-transpiler](https://github.com/libocca/occa-transpiler/blob/main/README.md)
```
Comment (IuriiKobein) [inline INSTALL.md:81]: Please keep the link to occa-trasnpiler README here as reminder to the user to install it.

**Review by IuriiKobein — CHANGES_REQUESTED** [review]
I don't have any objections to link a clang based transpiler as third party library.
The only thing is to provide all necessary information to the user how to build it before using it with OCCA.

File: INSTALL.md (hunk 1 above)
Comment (thilinarmtb) [reply INSTALL.md:81]: Fixed this in the latest commit.

**Review by IuriiKobein — APPROVED** [review]
The proper name should be clang based okl-transpiler to distinguish it from legacy one. 
And let align it later.

**Comment by thilinarmtb** [comment]
@IuriiKobein : Can you please merge this if this looks okay?

**Comment by thilinarmtb** [comment]
@IuriiKobein : Please let me know if you have additional comments.

**Comment by IuriiKobein** [comment]
@thilinarmtb No comments. PR is merged. Thanks.

**Comment by thilinarmtb** [comment]
Thank you very much !

**Pull request merged**